bin/phenomenal-post-install.sh

```diff
@@ -30,12 +30,27 @@ else
   echo "http://galaxy.$domain"
   echo "http://notebook.$domain"
   echo "http://luigi.$domain"
-  echo "http://dashboard.$domain"
+  # Checking if extra services have been enable or not. If so, then returning url
+  dashboard=$(grep "dashboard_include" config.tfvars | awk '{ print $3 }' | sed s/\"//g)
+  logmon=$(grep "logmon_include" config.tfvars | awk '{ print $3 }' | sed s/\"//g)
+  data_federation=$(grep "datafed_include" config.tfvars | awk '{ print $3 }' | sed s/\"//g)
+  
+  if [[ "$dashboard" == true ]]; then
+    echo "http://dashboard.$domain"
+  fi
+  if [[ "$logmon" == true ]]; then
+    echo -e "\nLogging and Monitoring Tools:\n"
+    echo "http://kibana.$domain"
+    echo "http://prometheus.$domain"
+    echo "http://grafana.$domain"
+  fi
+  if [[ "$data_federation" == true ]]; then
+    echo -e "\nData Federation Tool:\n"
+    echo "http://owncloud.$domain"
+  fi
```

Comment:
Here I took the liberty of printing out optional service's URL if and only if their flags are equal to "true"

Reply:
Thanks! Great!

Reply:
You could change the post_install check to use same style of reading params as in kn-apply, e.g.

# Read config file into variable as as json
kn_config=$(json2hcl -reverse <config.tfvars)

# Set variables from json-config
galaxy_include=$(jq -r '.galaxy_include' <<<"$kn_config")
luigi_include=$(jq -r '.luigi_include' <<<"$kn_config")
dashboard_include=$(jq -r '.dashboard_include' <<<"$kn_config")

and so forth...